cmd/buildah/export.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"github.com/containers/storage/pkg/archive"
```

Comment:
I think you're going to want to use the neighboring `chrootarchive` package here.